lib/plugins/after/getAuditPayload.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+/**
+ * Takes in a request and response and produces a payload of the req/res
+ * pair.
+ *
+ * @param {Object} req -
+ * @param {Object} res -
+ * @param {Object} err -
+ * @returns {Object} - The audit payload.
+ * @private
+ */
+module.exports = function getAuditPayload(req, res, err) {
```

Comment:
This should cache the audit payload onto the request object the first time, and use it thereafter in subsequent invocations.